### DISCUSSION TITLE
Git Integration

### DISCUSSION THREAD

**Discussion by AGSSAS** [TabularEditor/TabularEditor3#873] [Other Topics]
Is there any plan for when Git Integration is coming for Tabular Editor 3?

I see it on the "future" roadmap and that there was a private preview in 2022.  I think this would be tremendously helpful to have this integrated in the app and wonder if there is a more targeted date for this.  Thanks.

https://github.com/TabularEditor/TabularEditorDocs/blob/main/te3/other/roadmap.md/

**Comment by otykier** [comment]
It is still on our roadmap, but we have decided to prioritize other features over git integration, for the time being. The primary reason is, that a full-fledged git integration in TE3 (complete with diff and merge capabilities, etc.) is a big investment, and there are many other (free) tools available, that have these capabilities, and more. As such, we don't feel that git integration in TE3 would provide enough value to justify the cost, but we are open to debating this. If you send an e-mail to support@tabulareditor.com, we can also send you instructions on how to enable the private preview.
